libforget/misskey.py

```diff
@@ -0,0 +1,210 @@
+from app import db, sentry
+from model import MisskeyApp, MisskeyInstance, Account, OAuthToken
+from requests import get, post
```

Comment:
instead of using `requests.get()` and `requests.post()` directly, please use a requests [Session](https://docs.python-requests.org/en/latest/api/#requests.Session) from `libforget.session.make_session()`. it sets its user agent to something friendly and lets us reuse connections